### PR TITLE
Dedicated Host modules

### PR DESCRIPTION
**Pull request by tischrei**

### CODE REVIEW

**Comment by tischrei** [comment]
The UUIDs and other data aren't real values.

File: plugins/modules/deh_host_info.py [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+                    "yourtag": "yourvalue"
+                }
+          ]
+      }
+    ]
+'''
+
+EXAMPLES = '''
+# Query all DeH hosts
+- deh_host_info:
```
Comment (gtema) [inline plugins/modules/deh_host_info.py:131]: please use FQCN in examples

File: plugins/modules/deh_host_info.py [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+DOCUMENTATION = '''
+module: deh_host_info
+short_description: Get Dedicated host info
+extends_documentation_fragment: opentelekomcloud.cloud.otc
+version_added: "0.0.1"
```
Comment (gtema) [inline plugins/modules/deh_host_info.py:18]: version is too old

File: plugins/modules/deh_host_type_info.py [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+DOCUMENTATION = '''
+module: deh_host_type_info
+short_description: Get info about all available host types in a AZ
+extends_documentation_fragment: opentelekomcloud.cloud.otc
+version_added: "0.0.1"
```
Comment (gtema) [inline plugins/modules/deh_host_type_info.py:18]: version is too old

File: plugins/modules/deh_host_type_info.py [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+            "host_type_name": "General computing",
+            "id": null,
+            "name": null
+        }
+    ]
+'''
+
+EXAMPLES = '''
+# Query all host types in an Availability Zone
+- deh_host_type_info:
```
Comment (gtema) [inline plugins/modules/deh_host_type_info.py:78]: FQCN

File: plugins/modules/deh_server_info.py [hunk 5]
```diff
@@ -0,0 +1,162 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+DOCUMENTATION = '''
+module: deh_server_info
+short_description: Get info about ECSs on a Dedicated host
+extends_documentation_fragment: opentelekomcloud.cloud.otc
+version_added: "0.0.1"
```
Comment (gtema) [inline plugins/modules/deh_server_info.py:18]: version is too old

File: plugins/modules/deh_server_info.py [hunk 6]
```diff
@@ -0,0 +1,162 @@
...
+          "tenant_id": "16d53a84a13b49529d2e2c3646691123",
+          "updated_at": "2020-09-30T09:59:15Z",
+          "user_id": "18569c6d589c4be3a300b6401c74d123"
+      }
+    ]
+'''
+
+EXAMPLES = '''
+# Query all ECSs on DeH host
+- deh_server_info:
```
Comment (gtema) [inline plugins/modules/deh_server_info.py:113]: FQCN

**Review by gtema — CHANGES_REQUESTED** [review]

File: plugins/modules/deh_host_info.py (hunk 1 above)
Comment (tischrei) [reply plugins/modules/deh_host_info.py:131]: done

File: plugins/modules/deh_host_info.py (hunk 2 above)
Comment (tischrei) [reply plugins/modules/deh_host_info.py:18]: I don't understand your "version procedere". For me: The first version is the first edit. After the thing is released and I change something, I edit the version. What do you expect?

File: plugins/modules/deh_host_type_info.py (hunk 3 above)
Comment (tischrei) [reply plugins/modules/deh_host_type_info.py:18]: done

File: plugins/modules/deh_host_type_info.py (hunk 4 above)
Comment (tischrei) [reply plugins/modules/deh_host_type_info.py:78]: done

File: plugins/modules/deh_server_info.py (hunk 5 above)
Comment (tischrei) [reply plugins/modules/deh_server_info.py:18]: done

File: plugins/modules/deh_host_info.py (hunk 2 above)
Comment (gtema) [reply plugins/modules/deh_host_info.py:18]: This info is for users to be able to understand starting from which released version this module is available (when looking to docs, etc)

File: plugins/modules/deh_server_info.py (hunk 6 above)
Comment (tischrei) [reply plugins/modules/deh_server_info.py:113]: done

File: plugins/modules/deh_host_info.py (hunk 2 above)
Comment (tischrei) [reply plugins/modules/deh_host_info.py]: Okidoki, I have changed the versions again according to our current label.

**Comment by tischrei** [comment]
recheck

**Comment by tischrei** [comment]
recheck

**Comment by tischrei** [comment]
recheck

**Comment by tischrei** [comment]
@gtema all changes are done, all other modules are added and integration tests are done.

**Review by gtema — APPROVED** [review]

**Pull request merged**